Content.Tests/DMTests.cs

```diff
@@ -28,6 +30,40 @@ public void OneTimeSetup()
             _dreamMan.Initialize(SetupCompileDm.CompiledProject);
         }
 
+        [Test, TestCaseSource(nameof(GetTests))]
+        public void TestFiles(string file)
+        {
+            var prev = _dreamMan.DMExceptionCount;
+
+            string firstLine;
+
+            using(StreamReader reader = new StreamReader(file))
+            {
+                firstLine = reader.ReadLine() ?? "";
+            }
+
+            bool shouldRuntime = firstLine.Contains("RUNTIME TRUE", StringComparison.InvariantCulture);
+
+            var fileName = Path.GetFileName(file)[..^3]; // Ignore the ".dm" extension
+
+            var result = DreamThread.Run(async(state) => {
+                var world = _dreamMan.WorldInstance;
+                var proc = world.GetProc($"{fileName}_Proc");
+                return await state.Call(proc, world, null, new DreamProcArguments(null));
```

Comment:
This should check if the proc exists first.
```suggestion
                
                if (world.TryGetProc($"{fileName}_Proc", out DreamProc proc)) {
                    return await state.Call(proc, world, null, new DreamProcArguments(null));
                } else {
                    Assert.Fail($"No proc named {fileName}_Proc");
                    return DreamValue.Null;
                }
```